src/screens/TopicMemberScreen/index.js

```diff
@@ -228,13 +180,17 @@ const TopicMemberScreen = () => {
         onCancelToken={onTokenCancel}
         placeholderText={StringConstant.topicMemberPlaceholder}
         setIsFirstTimeOpen={setIsFirstTimeOpen}
-        hideBackIcon={true}
+        getSearchLayout={setSearchHeight}
       />
       <ScrollView
+        decelerationRate="fast"
+        scrollEventThrottle={1}
         style={styles.fragmentContainer}
-        contentContainerStyle={styles.fragmentContentContainer}
+        contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
         keyboardShouldPersistTaps="handled"
-        onScroll={handleScrollEvent}>
+        keyboardDismissMode="on-drag"
+        removeClippedSubviews={true}
+        onScroll={handleScroll}>
         <UsersFragment
           isLoadingDiscoveryUser={isLoadingDiscovery.user}
           isFirstTimeOpen={isFirstTimeOpen}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [173-184]

The `hasSearch` prop has been added to the `NavHeader` component, and the `getSearchLayout` prop has been added to the `Search` component. Ensure that these new props are properly handled in their respective components.
```diff
+ hasSearch={true}
+ getSearchLayout={setSearchHeight}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->